rust/chains/hyperlane-cosmos/src/providers/grpc.rs

```diff
@@ -534,15 +521,43 @@ impl WasmProvider for WasmGrpcProvider {
         Ok(response.data)
     }
 
+    async fn wasm_contract_info(&self) -> ChainResult<ContractInfo> {
+        let contract_address = self.get_contract_address()?;
+        let response = self
+            .provider
+            .call(move |provider| {
+                let to = contract_address.address().clone();
+                let future = async move {
+                    let mut client = WasmQueryClient::new(provider.channel.clone());
+
+                    let request = tonic::Request::new(QueryContractInfoRequest { address: to });
+
+                    let response = client
+                        .contract_info(request)
```

Comment:
I assume the response size here is generally pretty lightweight? `ContractInfo` seems modest https://docs.rs/terra-proto-rs/latest/terra_proto_rs/cosmwasm/wasm/v1/struct.ContractInfo.html but curious if the extension is ever very large

Reply:
The extensions are empty in the test I've done.